packages/contracts-bedrock/test/L2/OptimismSuperchainERC20.t.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.25;
+pragma solidity ^0.8.0;
```

Comment:
Why this diff?

Reply:
From when i was trying to get it to compile. Not necessary as i changed it in DeployUtils